### ISSUE TITLE
"constructor" constructs don't expose their arguments

### ISSUE THREAD

**Issue by tabatkins** [plinss/widlparser#11]
Given this IDL fragment:

```
[Constructor(long foo, optional long bar), 
 NamedConstructor=One, 
 NamedConstructor=Two(long foo, optional long bar)]
interface Bar {
	long baz(long foo, optional long bar);
};
```

The "method" construct for baz has an `arguments` attr when `markupName` is called for it, but none of the constructors do.

**Comment by plinss** [comment]
fixed in 9d1da169ef90472678a60a2ca9b203e132ae77bb

**Comment by plinss** [comment]
FWIW, the constructor construct is really an ExtendedAttribute which is a wrapper holding the real attribute in it's 'attribute' property. I added an 'arguments' property to the ExtendedAttribute which reflects the inner attribute's arguments.
